playground/app/components/todo-item.component.ts

```diff
@@ -12,9 +12,19 @@ export class TodoItemCmp implements OnChanges{
 
   @Input('<') todo: TodoModel;
   @Input('<') idx: number;
-  @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  // this is old way how to bindings are created, will be removed in 2.0
+  // @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  @Output() onDone: EventEmitter<TodoModel>;
```

Comment:
In contrary, you can do it your way and my way, both works ;) interface of new instance, everybody wins! ;)